index.d.ts

```diff
@@ -1,12 +1,49 @@
-import { Parser } from 'acorn' 
+import * as acorn from 'acorn';
 
-declare const jsx: (options?: jsx.Options) => (BaseParser: typeof Parser) => typeof Parser;
+export const jsx: (options?: jsx.Options) => (BaseParser: typeof acorn.Parser) => typeof AcornJsxParser
 
-declare namespace jsx {
-  interface Options {
-    allowNamespacedObjects?: boolean;
-    allowNamespaces?: boolean;
-  }
+export interface Options {
+  allowNamespacedObjects?: boolean;
+  allowNamespaces?: boolean;
 }
 
-export = jsx;
+export const tokTypes: {
+  jsxName: acorn.TokenType,
+  jsxText: acorn.TokenType,
+  jsxTagEnd: acorn.TokenType,
+  jsxTagStart: acorn.TokenType
+} & typeof acorn.tokTypes;
+
+export const tokContexts: {
```

Comment:
This declares `tokContexts` is available as a named variable, but it’s not.

Reply:
This one both makes sense and raises no questions. I've added a commit to amend my PR (and can continue to amend as I come to terms with the rest). Thanks!